src/data/heal_locations_pkm_center.h

```diff
@@ -0,0 +1,171 @@
+    {
+        .group = MAP_GROUP(DEWFORD_TOWN_POKEMON_CENTER_1F),
+        .map = MAP_NUM(DEWFORD_TOWN_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+    [HEAL_LOCATION_LAVARIDGE_TOWN - 1] =
+    {
+        .group = MAP_GROUP(LAVARIDGE_TOWN_POKEMON_CENTER_1F),
+        .map = MAP_NUM(LAVARIDGE_TOWN_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+    [HEAL_LOCATION_FALLARBOR_TOWN - 1] =
+    {
+        .group = MAP_GROUP(FALLARBOR_TOWN_POKEMON_CENTER_1F),
+        .map = MAP_NUM(FALLARBOR_TOWN_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+    [HEAL_LOCATION_VERDANTURF_TOWN - 1] =
+    {
+        .group = MAP_GROUP(VERDANTURF_TOWN_POKEMON_CENTER_1F),
+        .map = MAP_NUM(VERDANTURF_TOWN_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+    [HEAL_LOCATION_PACIFIDLOG_TOWN - 1] =
+    {
+        .group = MAP_GROUP(PACIFIDLOG_TOWN_POKEMON_CENTER_1F),
+        .map = MAP_NUM(PACIFIDLOG_TOWN_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+    [HEAL_LOCATION_EVER_GRANDE_CITY_POKEMON_LEAGUE - 1] =
+    {
+        .group = MAP_GROUP(EVER_GRANDE_CITY_POKEMON_LEAGUE_1F),
+        .map = MAP_NUM(EVER_GRANDE_CITY_POKEMON_LEAGUE_1F),
+        .x = 3,
+        .y = 4,
+    },
+    [HEAL_LOCATION_SOUTHERN_ISLAND_EXTERIOR - 1] =
+    {
+        .group = MAP_GROUP(SOUTHERN_ISLAND_EXTERIOR),
+        .map = MAP_NUM(SOUTHERN_ISLAND_EXTERIOR),
+        .x = 15,
+        .y = 20,
+    },
+    [HEAL_LOCATION_BATTLE_FRONTIER_OUTSIDE_EAST - 1] =
+    {
+        .group = MAP_GROUP(BATTLE_FRONTIER_POKEMON_CENTER_1F),
+        .map = MAP_NUM(BATTLE_FRONTIER_POKEMON_CENTER_1F),
+        DEFAULT_POKEMON_CENTER_COORDS,
+    },
+};
+
+#undef DEFAULT_POKEMON_CENTER_COORDS
+
+static const u8 sWhiteoutRespawnHealerNpcLocalIds[HEAL_LOCATION_COUNT - 1] =
+{
+    [HEAL_LOCATION_LITTLEROOT_TOWN_BRENDANS_HOUSE_2F - 1] = 1,
+    [HEAL_LOCATION_LITTLEROOT_TOWN_MAYS_HOUSE_2F - 1] = 1,
+    [HEAL_LOCATION_PETALBURG_CITY - 1] = 1,
+    [HEAL_LOCATION_SLATEPORT_CITY - 1] = 1,
+    [HEAL_LOCATION_MAUVILLE_CITY - 1] = 1,
+    [HEAL_LOCATION_RUSTBORO_CITY - 1] = 1,
+    [HEAL_LOCATION_FORTREE_CITY - 1] = 1,
+    [HEAL_LOCATION_LILYCOVE_CITY - 1] = 1,
+    [HEAL_LOCATION_MOSSDEEP_CITY - 1] = 1,
+    [HEAL_LOCATION_SOOTOPOLIS_CITY - 1] = 1,
+    [HEAL_LOCATION_EVER_GRANDE_CITY - 1] = 1,
+    [HEAL_LOCATION_LITTLEROOT_TOWN_BRENDANS_HOUSE - 1] = 1,
+    [HEAL_LOCATION_LITTLEROOT_TOWN_MAYS_HOUSE - 1] = 1,
+    [HEAL_LOCATION_OLDALE_TOWN - 1] = 1,
+    [HEAL_LOCATION_DEWFORD_TOWN - 1] = 1,
+    [HEAL_LOCATION_LAVARIDGE_TOWN - 1] = 1,
+    [HEAL_LOCATION_FALLARBOR_TOWN - 1] = 1,
+    [HEAL_LOCATION_VERDANTURF_TOWN - 1] = 1,
+    [HEAL_LOCATION_PACIFIDLOG_TOWN - 1] = 1,
+    [HEAL_LOCATION_EVER_GRANDE_CITY_POKEMON_LEAGUE - 1] = 1,
+    [HEAL_LOCATION_SOUTHERN_ISLAND_EXTERIOR - 1] = 0, // no heal npc
+    [HEAL_LOCATION_BATTLE_FRONTIER_OUTSIDE_EAST - 1] = 1,
```

Comment:
When I get warped here when I lose, I get warped OUTSIDE of the pokemon center but the nurse joy script still plays. Any ideas why?

Reply:
Not sure, didn't observer that while testing myself.